src/GraphControl/Control/Grapher.h

```diff
@@ -20,13 +20,16 @@ public
 
 public
     delegate void TracingValueChangedEventHandler(Windows::Foundation::Point value);
+public
+    delegate void PointerValueChangedEventHandler(Windows::Foundation::Point value);
 
     [Windows::UI::Xaml::Markup::ContentPropertyAttribute(Name = L"Equations")] public ref class Grapher sealed
         : public Windows::UI::Xaml::Controls::Control,
           public Windows::UI::Xaml::Data::INotifyPropertyChanged
     {
     public:
         event TracingValueChangedEventHandler ^ TracingValueChangedEvent;
+        event PointerValueChangedEventHandler ^ PointerValueChangedEvent;
```

Comment:
>PointerValueChangedEvent [](start = 48, length = 24)

passing the trace point back via event.  Does this have perf implications?  It seems ok on my box, but if there's a better way to do this, I'm all ears :)